### DISCUSSION TITLE
please support gate wallet

### DISCUSSION THREAD

**Discussion by yzhi3166** [rainbow-me/rainbowkit#1651] [Q&A]
please support gate wallet，thank you！！！

**Comment by 0xmkh** [comment]
@yzhi3166 Just had a look at gate wallet and it seems like they have some limitations based on the regions. They do not allow UK people to use the app itself and it's really important for us otherwise if we support gate wallet and the app can't be accessed due to region restrictions it would be a bad UX with WalletConnect mobile linking.

Let me know if you find any other wallets that doesn't have region restrictions like this 👍
